include/config/map_name_popup.h

```diff
@@ -0,0 +1,23 @@
+#ifndef GUARD_CONFIG_MAP_POPUP_H
+#define GUARD_CONFIG_MAP_POPUP_H
+
+// General map pop-up config
+#define MAP_POPUP_GENERATION        GEN_3   // Different generations display location names in overworld pop-ups differently
+                                            // only current other choice is GEN_5, all others will default to gen 3 pop-ups
```

Comment:
```suggestion
                                            // only current other choice is GEN_5, all others will default to Gen3 pop-ups
```
Matter of consistency with the wording of other configs